exercises/practice/triangle/triangle_test.rb

```diff
@@ -86,28 +86,40 @@ def test_isosceles_triangle_sides_may_be_floats
     assert triangle.isosceles?, "Expected 'true', triangle [0.5, 0.4, 0.5] is isosceles."
   end
 
-  def test_triangle_is_scalene_if_no_sides_are_equal
+  def test_scalene_triangle_no_sides_are_equal
     skip
     triangle = Triangle.new([5, 4, 6])
     assert triangle.scalene?, "Expected 'true', triangle [5, 4, 6] is scalene."
   end
 
-  def test_triangle_is_not_scalene_if_all_sides_are_equal
+  def test_scalene_triangle_all_sides_are_equal
     skip
     triangle = Triangle.new([4, 4, 4])
-    refute triangle.scalene?, "Expected 'false', triangle [4, 4, 4] is not scalene."
+    refute triangle.scalene?, "Expected 'false', triangle [4, 4, 4] is scalene."
   end
 
-  def test_triangle_is_not_scalene_if_two_sides_are_equal
+  def test_scalene_triangle_first_and_second_sides_are_equal
     skip
     triangle = Triangle.new([4, 4, 3])
-    refute triangle.scalene?, "Expected 'false', triangle [4, 4, 3] is not scalene."
+    refute triangle.scalene?, "Expected 'false', triangle [4, 4, 3] is scalene."
```

Comment:
If we are refuting that triangle.scalene?, then isn't it false, because 4, 4, 3 is not scalene?  Otherwise, if it is scalene, t hen scalene? would not be refuted, but it would be true.

Reply:
My bad! Thanks for catching this.